### PR TITLE
update NodeJs / ExpressJs Tutorial

### PR DESCRIPTION
**Pull request by ilyaschicha**
This update aims to provide a more user-friendly and comprehensive experience for readers following the tutorial. The changes have been thoroughly reviewed and tested to ensure accuracy. Your feedback and suggestions are welcome to further improve the tutorial's quality and usefulness.

### CODE REVIEW

File: tutorials/node.js/express.js.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+![](../../.gitbook/assets/WebsitesAndDomains.png)
+
+Under the domain you'd like to use for Node.js, select `Node.js`.
+
+![](../../.gitbook/assets/NodejsLink.png)
+
+## Server setup
+Upload all your files to your Tommy server. Then go to the Node.js section and set the following.
+
+* Node.js Version: At least 14.21.0 tho i highly recommend using the latest one 17.9.1
```
Comment (BaileyJM02) [inline tutorials/node.js/express.js.md]:
```suggestion
* Node.js Version: minimum 14.21.0, though it is recommend to use the latest available, currently 17.9.1.
```

File: tutorials/node.js/express.js.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* Application Root: / (folder where you uploaded your next project, i recommend / )
```
Comment (BaileyJM02) [inline tutorials/node.js/express.js.md]:
```suggestion
* Application Root: `/` (This is the folder where you uploaded your project)
```

File: tutorials/node.js/express.js.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* Application Startup File: app.js (file we created earlier)
```
Comment (BaileyJM02) [inline tutorials/node.js/express.js.md]:
```suggestion
* Application Startup File: `app.js` (The entry point of our app, use the file we created earlier)
```

File: tutorials/node.js/express.js.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+Then you need to press the NPM install button.
+
+![](../../.gitbook/assets/NPMInstall.png)
+
+After that finished you just need to wait for the server to update your account and start using Node.js, this may take a while.
+
+In order for the Node.js application to be deployed, it requires an Apache restart, so this could take anywhere from a few minutes to a few hours. If it's been more than 2 hours, and it still isn't working, please
+open [a customer service ticket](https://www.helionet.org/index/forum/45-customer-service/) and let us know.
+
+Once the Node.js application has been deployed you should be able to go to `domain.helioho.st/` and see the
```
Comment (BaileyJM02) [inline tutorials/node.js/express.js.md]:
```suggestion
Once the Node.js application has been deployed you should be able to go to the domain you selected and see the
```

**Review by BaileyJM02 — CHANGES_REQUESTED** [review]
Looks good so far, a few small changes

File: tutorials/node.js/README.md
Comment (BaileyJM02) [inline tutorials/node.js/README.md:1]: Could you revert the changes to this file? This is a useful bare-bones example of a NodeJS integration.

Comment (ilyaschicha) [reply tutorials/node.js/README.md:1]: @BaileyJM02  sure i will return it to it's original content one thing is when i used gitbook i found that you can access the folder Node.js without knowing that there is content inside it i realize that there is content inside only when Kryd0s send me the link for this repository

**Review by ilyaschicha — COMMENTED** [review]
i return it to it's original form but i update the images and add some images to the content

**Comment by Kryd0s** [comment]
Thanks for the contribution.

**Comment by ilyaschicha** [comment]
do you want me make tutorial of using Git in plesk @Kryd0s

**Pull request merged**